io/src/lib.rs

```diff
@@ -6,9 +6,12 @@ use scale_info::TypeInfo;
 
 #[derive(Debug, Default, Encode, Decode, TypeInfo, Clone)]
 pub struct LotteryState {
```

Comment:
Why is it necessary to take it out in a separate structure?

Reply:
to send in the meta_state() function